fe1-web/src/features/social/navigation/SocialMediaNavigation.tsx

```diff
@@ -50,58 +48,62 @@ const iconSelector =
 const SocialMediaNavigation = () => {
   const [currentUserPublicKey, setCurrentUserPublicKey] = useState(new PublicKey(''));
 
-  const lao = useSelector(selectCurrentLao);
+  const lao = SocialHooks.useCurrentLao();
 
   if (lao === undefined) {
     throw new Error('LAO is currently undefined, impossible to access to Social Media');
   }
 
   // Get the pop token of the user using the last tokenized roll call
   const rollCallId = lao.last_tokenized_roll_call_id;
-  const eventSelect = makeRollCallSelector(rollCallId);
-  const rollCall: RollCall = useSelector(eventSelect) as RollCall;
+  if (rollCallId === undefined) {
+    throw new Error(
+      'Last tokenized roll call id is undefined, impossible to access to Social Media',
+    );
+  }
```

Comment:
I don't think the order/number of hooks is an issue there, because if this id is undefined, the app will crash anyway because of the Error.

I agree with you that it's not the perfect thing to do. Actually, I think that the Social Media shouldn't be accessible if no roll call happened within the LAO. In my opinion, the best would be to not show the social media button in the bottom navigation bar but since a lot of UI has changed I don't know how much time it will take for me to do so. Another solution would be to replace the whole social media navigation by a simple page with a message telling that it's unavailable if you don't have a token.
What do you think?

Reply:
Actually after the changes every page of the social media can be opened because the `currentUserPublicKey` is just undefined, so I guess it's ok for now. Obviously, you can't perform any action because you don't have a profile yet.